it.lproj/Localizable.strings

```diff
@@ -191,10 +191,10 @@
 "Achievements" = "Successi";
 
 /* title for the Day Total awards section of the Achievements area */
-"Day Totals" = "Totali giornalieri";
+"Day Totals" = "Passi giornalieri";
```

Comment:
This section includes the number of the user's daily steps. "Passi giornalieri" equals "Daily steps"